src/Client.php

```diff
@@ -117,22 +127,18 @@ public function getVersion(bool $without_hash = true): string
     /**
      * {@inheritdoc}
      */
-    public function devToken(string $username,
-                             string $password,
-                             bool $is_hash = false,
-                             ?DateTime $date_from = null,
-                             int $age = 60): DevTokenResponse
+    public function devToken(DevTokenParams $params): DevTokenResponse
     {
         return DevTokenResponse::fromHttpResponse(
             $this->doRequest(new Request('get', 'dev/token'), [
                 'query' => [
-                    'user'    => $username,
-                    'pass'    => $password,
-                    'is_hash' => $is_hash
+                    'user'    => $params->getUsername(),
+                    'pass'    => $params->getPassword(),
+                    'is_hash' => $params->isPasswordHashed()
```

Comment:
```suggestion
                    'is_hash' => $params->isPasswordHashed() === true
```

Here and in all places, where method can return `nullable` value

Reply:
I went through the code. This is the only such place where a more stringent condition is required.

Fixed 166276ba317b164c336c5ff7ce3357a53a49fb74